docs/metrics/accelerate.md

```diff
@@ -81,6 +81,8 @@ Pulse uses the average value over a period of time to display aggregate change f
 
     As such, Pulse considers the **deployment** that caused an **incident** the closest deployment before the start of that **incident**. The same deployment might be responsible for multiple incidents.
 
+    When viewing the performance, it is calculated per system before being aggregated by the time buckets. This also means that you need the deployments and incidents to have the right system association for the metrics to work.
+
```

Comment:
"Time buckets" is internal lingo that customers may not understand. Can we use "time intervals" instead?

Also, here's my go at trying to improve the readability of this information by making some details more explicit:

```suggestion
    Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.
```

Reply:
🎖️

Reply:
Intervals sounds weird to me since that is the other thing in the filters. The buckets are the aggregation intervals at most.

Reply:
OK, so maybe we can call them "time aggregation intervals" - or even "time aggregation buckets"? At least for me, this already clarifies better how things work. :ok_hand: